it/test/controllers/StartPageControllerISpec.scala

```diff
@@ -121,7 +120,7 @@ class StartPageControllerISpec extends IntegrationTest with ViewHelpers {
   import CommonExpectedResults._
 
   private val urlPath = s"/update-and-submit-income-tax-return/$taxYear/start"
-
+  private val tailoringPhase2UrlPath = s"http://localhost:10007/update-and-submit-income-tax-return/tailored-return/$taxYear/start"
```

Comment:
We can use the config value here instead of hard coded string, could also add types to the values.
`s"${frontendAppConfig.tailorReturnServiceUrl}/$taxYear/start"`